rpc/server_auth.go

```diff
@@ -404,11 +382,17 @@ func (ss *simpleServer) ensureAuthed(ctx context.Context) (context.Context, erro
 
 	err = claims.Valid()
 	if err != nil {
+		ss.logger.Errorw("invalid claims",
+			"error", err,
+			"registeredClaims", claims.RegisteredClaims)
```

Comment:
nit: registered_claims (snake case) for structured fields

Reply:
thank you! love these kinds of nits :)